src/main/java/edu/hm/hafner/analysis/parser/dry/cpd/CpdParser.java

```diff
@@ -64,6 +64,8 @@ protected Report convertDuplicationsToIssues(final List<Duplication> duplication
                         .setLineStart(file.getLine())
                         .setLineEnd(file.getLine() + duplication.getLines() - 1)
                         .setFileName(file.getPath())
+                        .setCategory("CPD")
+                        .setType("CodeDuplication")
```

Comment:
And use „CPD“ as type here. It would be nice to set the type for the other two parsers as well.